wire-ios-sync-engine/Source/UserSession/ZMUserSession+CoreCrypto.swift

```diff
@@ -35,10 +35,12 @@ extension ZMUserSession {
         }
 
         switch stage {
-        case .proteus(let userID):
+        case .proteus(let userID) where shouldSetupProteusService:
```

Comment:
I think we should rename these to `shouldSetUpProteus` and `shouldSetUpMLS` since these flags were previous used to conditionally create the controllers, but now they are more general.